### ISSUE TITLE
unable to import PKCS#8 objects with short password

### ISSUE THREAD

**Issue by dkg** [openssl/openssl#16231]
This pem-encoded PKCS#8 object: [bob.p8.txt](https://github.com/openssl/openssl/files/6939207/bob.p8.txt) has a password that is the simple three-letter ASCII string `bob`.

I'm unable to read it with OpenSSL:

```
$ openssl pkey -inform pem < bob.p8.txt
Enter pass phrase:
140622091945280:error:28078065:UI routines:UI_set_result_ex:result too small:../crypto/ui/ui_lib.c:905:You must type in 4 to 1023 characters
```

While the limit of 4 character passwords might be arguably reasonable to impose on pkcs#8 *generation* it is unreasonable to impose that constraint when trying to interpret incoming data.

**Comment by t8m** [comment]
This is 1.1.1 only issue.

**Comment by t8m** [comment]
Merged to 1.1.1 branch. Thank you.
